tsweb/tsweb.go

```diff
@@ -382,6 +383,9 @@ func varzHandler(w http.ResponseWriter, r *http.Request) {
 				dump(name+"_", kv)
 			})
 			return
+		case *expvar.String:
+			fmt.Fprintf(w, "# TYPE %s literal\n%s %v\n", name, name, v.Value())
```

Comment:
Where is this documented?

Not at https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md (as referenced on line 341 above)

What's the quoting rules for the string?

Reply:
Oops. I was looking at the querying docs. :( Thanks.